packages/common/src/deserialization.ts

```diff
@@ -202,3 +202,37 @@ export function deserializeTransaction(
             throw new Error('Invalid blockItemKind');
     }
 }
+
+/**
+ * Deserializes a return value from a sequence of bytes into a json object.
+ * @param returnValueBytes A buffer containing the return value as raw bytes.
+ * @param moduleSchema The raw module schema as a buffer.
+ * @param contractName The name of the contract where the receive function is located.
+ * @param functionName The name of the receive function whose return value you want to deserialize.
+ * @param schemaVersion The schema version as a number. This parameter is optional,
+ * if you provide a serialized versioned schema this argument won't be needed.
+ */
+export function deserializeReturnValue(
```

Comment:
Please add an section/example to the common package's README of using this function. (Just as short as it need to be, you can check the deserialize state for reference)
+ Add an entry to the changelog. Just put it under version 5.2.0.